.docker_utils/main_dock.sh

```
@@ -19,6 +19,8 @@ docker run --tty \
     --volume $FORZETH_DIR/../cache/humble/log:/home/$USER/ws/log \
     --volume $FORZETH_DIR:/home/$USER/ws/src/race_stack \
     --privileged \
+    --group-add $(getent group input | cut -d: -f3) \
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)